app/helpers/service_calendar_helper.rb

```diff
@@ -136,6 +136,12 @@ def display_total_cost_per_study_otfs service_request, line_items
   #############################################
   # Grand Totals
   #############################################
+  def display_ssr_grand_total sub_service_request
+    sum = 0
```

Comment:
I'm not 100% sure, but if you look at the file, it's written the same way all the other methods there are, I was going for consistency (and they all need revisited anyways, as another story).